profiling-ffi/src/exporter.rs

```diff
@@ -169,6 +170,28 @@ unsafe fn into_vec_files<'a>(slice: Slice<'a, File>) -> Vec<exporter::File<'a>>
         .collect()
 }
 
+#[repr(C)]
+pub enum ExporterRequestBuildResult {
+    Ok(Box<Request>),
```

Comment:
On a second pass through the PR, I noticed that this is exposing a `ddog_prof_Box_Request` to the FFI, instead of a `ddog_prof_Exporter_Request *`.

This will probably cause issues since the other APIs expect `ddog_prof_Exporter_Request *` as input.